README.md

```diff
@@ -62,6 +62,9 @@ If you are reading this it looks like you are looking to add an egg to your serv
 * [MariaDB](/database/sql/mariadb)
 * [PostgreSQL](/database/sql/postgres)
 
+### Timeseries Databases
+* [Prometheus](/database/tsdb/prometheus)
```

Comment:
Prometheus is a monitoring tool

Reply:
Well, Prometheus is marketed more or less as being a "Monitoring system & time series database". But I will move it to an `monitoring` directory as it fits the purpose of Prometheus way better (and its actually the use-case I use Prometheus for). 

A `monitoring `directory does not really fit into the database folder neither into the `software` directory.
Where would you suggest to place that directory?

Reply:
make a monitoring folder if anything.

Reply:
Prometheus has been moved to a dedicated `monitoring` directoryy. The main README has also been updated.